rdr_service/dao/participant_summary_dao.py

```diff
@@ -1271,9 +1271,9 @@ def build_default_obj_response(self, obj: ParticipantSummary):
             if result.get(field_name) is None:
                 result[field_name] = UNSET
             elif result[field_name]:
-                result[field_name] = 'True'
+                result[field_name] = True
             else:
-                result[field_name] = 'False'
+                result[field_name] = False
```

Comment:
These are coming from the database as booleans, right? It seems like it'd be good to keep it simple by removing the elif and else, just changing the value if it's None. What do you think?